src/fcitx-rime.c

```diff
@@ -364,6 +365,15 @@ INPUT_RETURN_VALUE FcitxRimeGetCandWord(void* arg, FcitxCandidateWord* candWord)
     return retVal;
 }
 
+boolean FcitxRimePaging(void* arg, boolean prev) {
+    FcitxRime *rime = (FcitxRime *)arg;
+    boolean result = RimeProcessKey(rime->session_id, prev ? FcitxKey_Page_Up : FcitxKey_Page_Down, 0);
```

Comment:
According to https://github.com/rime/librime/blob/develop/src/gear/selector.cc#L32, XK_Prior/XK_Page_Up always do page up and  XK_Next/XK_Page_Down always do page down.
But maybe key binding works before selector?

@lotem Any suggestions to do page up/down? Add new API or just use RimeProcessKey?